pkg/config/validation/validation.go

```diff
@@ -3024,6 +3024,14 @@ func validateHTTPRedirect(redirect *networking.HTTPRedirect) error {
 	return nil
 }
 
+func validateHTTPDirectResponse(directResponse *networking.HTTPDirectResponse) error {
+	if directResponse == nil {
```

Comment:
can we add validation for the size? 

I would give a warning if its >100kb with something like "large responses may impact control plane performance" and then hard failure >1mb

Reply:
done!